assets/payment/settings_script.js

```diff
@@ -0,0 +1,286 @@
+/* global settings_object */
+/* exported redirectToCallback */
+let BuyLicenseWindow = null;
+let UpdatePlanWindow = null;
+let CancelSubscriptionWindow = null;
+let wait = false;
+/* eslint-disable */
+function redirectToCallback(param) {
+	window.location.href += "&" + param;
+}
+function cancelPayment(){
```

Comment:
Codacy found an issue: ['cancelPayment' is defined but never used.](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5907666)